get-involved.php

```diff
@@ -56,14 +56,15 @@
             <h2>General Funding</h2>
             <p>Patreon works like an ongoing crowdfunding campaign. Choose an amount, get rewards, and help us reach our goals. <a class="read-more" href="https://www.patreon.com/" target="blank">Learn More</a></p>
             <p>PayPal is a quick and easy solution. Choose a one-time amount or set up a subscription.</p>
-
+            <p>Buying merchandise directly supports the development of Elementary, making it better for everyone.</p>
```

Comment:
Just for completeness-sake for anyone looking at this later, on #1467 I commented that we could change this to 

> Buying merch from our store is another way to help us financially, plus you can get some exclusive elementary gear to show your support to friends, family, and coworkers.